steps/run/step.yaml

```diff
@@ -20,13 +20,26 @@ schemas:
     file: spec.schema.json
 
 examples:
-- summary: Start another workflow run
+- summary: Start the workflow `my-workflow` with `parameters`.
   content:
     apiVersion: v1
     kind: Step
-    name: workflow-run
+    name: my-workflow
     image: relaysh/workflow-step-run
     spec:
       name: workflow-b
       parameters:
         message: "a message for workflow b to print"
+- summary: Run the workflow `my-workflow` in a loop for every `parameters` 
```

Comment:
Using long unquoted strings in YAML gets really weird (like you can't use ":" characters), so I'd suggest making this a multi-line form using `>` instead.